packages/@okta/i18n/src/properties/login_ja.properties

```diff
@@ -305,7 +307,7 @@ email.mfa.email.sent.description.emailCodeText = \u30E1\u30FC\u30EB\u3092\u78BA\
 
 
 # Enroll Choices
-enroll.choices.title = \u30DE\u30EB\u30C1\u30D5\u30A1\u30AF\u30BF\u30FC\u8A8D\u8A3C\u3092\u30BB\u30C3\u30C8\u30A2\u30C3\u30D7\u3057\u307E\u3059
+enroll.choices.title = \u591A\u8981\u7D20\u8A8D\u8A3C\u3092\u30BB\u30C3\u30C8\u30A2\u30C3\u30D7\u3057\u307E\u3059
```

Comment:
same question here, since this is not changed in other languages. Was this a translation fix?

Reply:
Translation fix breadcrumb related to Japan Office feedback Japanese MFA terminology...